ci-scripts/rhdh-setup/deploy.sh

```diff
@@ -90,7 +90,7 @@ keycloak_install() {
 # shellcheck disable=SC2016,SC1004
 backstage_install() {
     until envsubst <template/backstage/secret-rhdh-pull-secret.yaml | $clin apply -f -; do $clin delete secret rhdh-pull-secret; done
-    envsubst '${RHDH_NAMESPACE} ${OPENSHIFT_APP_DOMAIN}' <template/backstage/app-config.yaml >app-config.yaml
+    envsubst '${RHDH_NAMESPACE} ${OPENSHIFT_APP_DOMAIN} ${KEYCLOAK_CLIENT_SECRET}' <template/backstage/app-config.yaml >app-config.yaml
```

Comment:
IMHO. you should not substitue this placeholder here (that is intentionally left intact). It is actually done by backstage itself based on the env variables taken and set from the actual Secret here: https://github.com/redhat-performance/backstage-performance/pull/24/files#diff-6090db30fa0bd300333e0cabff3cdf4a705bf1a61a534bc1ab3c1ec7ff251626L61

Reply:
It is a good practice to avoid hard-coding any secrets. IMHO, since you have that secret already stored in Secret resource in OpenShift, you should consume that secret from that if you can instead of putting it directly to the config. Such as in our case.

Reply:
My bad.. thanks for pointing it out. Let me resubmit with it fixed.